ortools/sat/python/cp_model.py

```diff
@@ -1634,7 +1638,7 @@ def AddModuloEquality(self, target, var, mod):
         model_ct.int_mod.target.CopyFrom(self.ParseLinearExpression(target))
         return ct
 
-    def AddMultiplicationEquality(self, target, *expressions):
+    def AddMultiplicationEquality(self, target, *expressions) -> Constraint:
         """Adds `target == expressions[0] * .. * expressions[n]`."""
```

Comment:
Should we correct this to reflect the fact that MultiplicationEquality will fail if `len(expressions) > 2` ?

Reply:
At some point, we will support it. So I would say no.